quinn-proto/src/coding.rs

```diff
@@ -1,18 +1,25 @@
+//! Coding related traits.
+
 use std::net::{Ipv4Addr, Ipv6Addr};
 
 use bytes::{Buf, BufMut};
 use thiserror::Error;
 
 use crate::VarInt;
 
+/// Unexpected end of buffer error.
```

Comment:
Try to use different terms than those in the item's name to help people for whom the name isn't self-explanatory.
```suggestion
/// Error indicating that the provided buffer was too small
```